tests/e2e/basic_test.go

```diff
@@ -193,8 +194,13 @@ func (s *BasicTestSuite) TestVmGroupVolumeIsolation(c *C) {
 	out, err = dockercli.DeleteVolume(s.vm1, s.volName2)
 	c.Assert(err, IsNil, Commentf(out))
 
+	// Clean up the vm group
+	out, err = admincli.DeleteVMgroup(s.esx, vmgroup)
+	c.Assert(err, IsNil, Commentf(out))
+
```

Comment:
I thought admincli does not returns error so this check does not work. Shouldn't we check if vmgroup was actually deleted to be sure that admincli.DeleteVMgroup() actually deleted the vmgroup?